[Email Routing] SPF Validation Clarification (#4500)
cloudflare/cloudflare-docs: content/email-routing/postmaster.md

Handling spam and abusive traffic is essential to any email provider. Below is a list of how Email Routing tackles this issue:

- Email Routing provides reverse DNS PTR records to all of its SMTP egress ranges. PTR records are often used as a reputation parameter.

> - Email Routing rejects emails whose SPF validation fails (hard fail only).

Reviewer: ```suggestion
- Email Routing rejects emails whose SPF validation results in a hard fail.
```
